pkg/virt-handler/virtwrap/schema.go

```diff
@@ -163,6 +169,33 @@ type DiskSourceHost struct {
 
 // END Disk -----------------------------
 
+// BEGIN Serial -----------------------------
+
+type Serial struct {
+	Type         string       `xml:"type,attr"`
+	SerialTarget SerialTarget `xml:"serialTarget"`
```

Comment:
I think the xml tag needs to be `xml:"target"`.

Also consider renaming the field name to`Target`.